general/administration/hardware-acceleration.md

```diff
@@ -49,6 +49,117 @@ The hardware acceleration is available immediately for media playback. No server
+```
+This is the ouput you are looking for
+`01:00.3 Serial bus controller [0c80]: NVIDIA Corporation TU116 [GeForce GTX 1650 SUPER] (rev a1)`
+Now check the firmware revision its on by typing ```ndvidia-smi```
+you should get an ouput like this
+Sun Aug 16 20:42:25 2020       
++-----------------------------------------------------------------------------+
+| NVIDIA-SMI 450.57       Driver Version: 450.57       CUDA Version: 11.0     |
+|-------------------------------+----------------------+----------------------+
+| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
+| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
+|                               |                      |               MIG M. |
+|===============================+======================+======================|
+|   0  GeForce GTX 165...  Off  | 00000000:01:00.0  On |                  N/A |
+|  0%   50C    P0    26W / 100W |    719MiB /  3895MiB |     17%      Default |
+|                               |                      |                  N/A |
++-------------------------------+----------------------+----------------------+
+
+(you specifically just need this to see if you have the latest available driver and if so then skip steps 9.-14.
+
+2. So to find the correct drivers select from the dropdown. Againb using my Geoforce 970 as an example
+        Product Type: Geoforce (whoever manufacters your GPU)
+        Product Series: Geoforce 900 Series (make sure you are not selecting the M unless your on a laptop)
+        Product Geoforce GTX 970 (Select what model of graphics card you have for your GPU)
+        Operating System: LINUX 64BIT (This is very important!!)        
+        Download Type: Select Linux Short Lived if your graphics card is a bit older.WHEN I SAY OLDER I MEAN 2 YEARS OR LESS.
+                       Select Linux Long Lived if your NVIDIA CARD IS FAIRLY NEW.
+        Click Search. This will automatically tell you your most updated driver you will need. Write or type this out somewhere so you can access it later.
+
+3. Now go to this link [NVIDIA Github patch](https://github.com/keylase/nvidia-patch) for patching the drivers.
+
+4. Scroll Down to the versions table or type in browser CTRL + F. Then type in your browser window for your drive
+        IE. Geoforce GTX 970 = 432.21 (it showed us the driver on nvidias website)
+
+5. Once youve located your correct driver hover over the driver link with your mouse right click the link and copy the link location.
+
+6. IMPORTANT!!! YOU NEED TO RUN THESE COMMANDS IN CLI NOT DE NOT DE TERMINAL CLI 
+        To do this we need to go out of GUI(Desktop Enviroment) and into CLI(kernal recovery terminal). 
+        So we do this by Pressing CTRL+ALT+F1 (Some people DE(desktop enviroment) will already be on F1 so change it to any other F#
+
+7. Now your should see a terminal login with your user
```

Comment:
I think you can assume that a user knows the try or is at least able to Google it

Reply:
i think it important for most new user or semi iregular users to be able to find this in the first link they load up. TBH o diodnt even know this until i was like 3-4 months working with linux. its a good refernce none the less. i feel its a bit important especially for the novice user. that doesnt understand

Reply:
Well a novice user really shouldn't patch his graphics driver, that can easily brick his system so it should only used my advanced users where possible. In either case if you write "open the tty" someone how hasn't encountered the tty will probably open something like "open the tty" which gives him exactly what he needs:
![Screenshot_20200817-124254.jpg](https://user-images.githubusercontent.com/26320625/90387980-6bae1900-e087-11ea-94de-5cf40ad7d49f.jpg)

Reply:
okay will do ill change that right now. Thank you i couldnt think of the name of it. I know it more as the of F**K recovery menu lol